src/Models/GFlasso.cpp

```diff
@@ -97,15 +97,15 @@ MatrixXd Gflasso::get_X(){
 }
 
 MatrixXd Gflasso::get_Y(){
-    return Y;
+    return y;
 };
 
 MatrixXd Gflasso::get_beta() {
     return beta;
 }
 
 void Gflasso::assertReadyToRun() {
-    throw runtime_error("Gflasso not implemented");
+    //throw runtime_error("Gflasso not implemented");
```

Comment:
Ok. Can you remove the useless interfaces (or comment them out), and add checks in the assertReadyToRun() section to make sure the necessary matrices are set?